containers/deploy/foreman/routes.yml

```diff
@@ -7,17 +7,21 @@
           apiVersion: v1
           kind: Route
           metadata:
-              name: foreman-8080
+              name: foreman-443
               namespace: "{{ project_name }}"
               labels:
                   app: foreman
-                  service: foreman
+                  service: httpd
           spec:
+              host: "foreman.{{ lookup('pipe', 'minishift ip') }}.nip.io"
```

Comment:
Ah, so this would be the minishift specific stuff you were talking about right?

Can we make this a variable and maybe default to this? That way we can set the hostname from the playbook if we're deploying to a cluster.

Reply:
Good catch. With the latest routing pattern I actually don't need to set host at all. So I'll remove this and let it be handled naturally.